config/routes.rb

```diff
@@ -3,6 +3,9 @@
 Rails.application.routes.draw do
   resources :posts do
     resources :reactions, only: %i[create update destroy]
+    resources :comments, only: %i[create update destroy] do
+      resources :replies, only: %i[create update destroy], controller: 'comments'
+    end
```

Comment:
If can, refactor this route to avoid deeply nested in route([Resources should never be nested more than 1 level deep](https://guides.rubyonrails.org/routing.html#limits-to-nesting)), you can have some preview about the shallow nesting

https://guides.rubyonrails.org/routing.html#shallow-nesting